auth/README.md

```diff
@@ -147,6 +148,14 @@ startActivityForResult(
     RC_SIGN_IN);
 ```
 
+<a href="https://developer.android.com/reference/android/app/Activity.html#startActivityForResult(android.content.Intent, int)">`startActivityForResult`</a>
+is part of the Android framework and takes two parameters: an Intent to start an activity--in this case,
+the sign in builder--and a request code of type `int`. For example, `REQUEST_CODE_SIGN_IN` or `RC_SIGN_IN`
+for short is a constant defined in our activity: `private static final int RC_SIGN_IN = 100;`.
+The number you chose is arbitrary as long as it doesn't conflict with other
+`startActivityForResult` requests you might be making. In [response codes](#response-codes),
+you will see how we can use `RC_SIGN_IN` to differentiate between multiple `startActivityForResult` requests.
+
```

Comment:
I think that discusses the mechanics of startActivityForResult a little too much, how about something like this:

To kick off the FirebaseUI sign in flow, call `startActivityForResult(...)` on the sign in Intent you built.  The second parameter (`RC_SIGN_IN`) is a request code you define to identify the request when the result is returned to your app in `onActivityResult(...)`.  See the [response codes](#response-codes) section below for more details on receiving the results of the sign in flow.